Matrix.C

```diff
@@ -704,7 +740,7 @@ Matrix::print(const char * prefix)
 {
    int my_rank;
    const bool success = MPI_Comm_rank(MPI_COMM_WORLD, &my_rank);
-   CAROM_VERIFY(success);
+   CAROM_ASSERT(success);
```

Comment:
What is the issue with `CAROM_VERIFY` here?

Reply:
For some reason, the verify was failing even though the rank was successfully retrieved, so I just changed it to an assert.